### PR TITLE
101 fix broken styling ngx

### PR DESCRIPTION
**Pull request by RensNooren**
## Description

Replaced Clarity with ngx bootstrap.
Fixed styling that got broken after replacing Clarity and fixed other styling problems.
Accordion styling that currently is implemented in the global scss will be fixed in a later issue.

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [X] My code follows the style guidelines of this project
-   [X] I did not update API Controllers, if I did, I added/changed Postman tests
-   [X] I have performed a self-review of my own code
-   [X] I have commented my code, particularly in hard-to-understand areas
-   [X] I have made corresponding changes to the documentation
-   [X] I have added tests that prove my fix is effective or that my feature works
-   [X] New and existing unit tests pass locally with my changes
-   [X] I updated the changelog with an end-user readable description
-   [X] I assigned this pull request to the correct project board to update the sprint board

## Link to issue

Closes: #101 #94

### CODE REVIEW

File: src/app/modules/project/add/manual/manual.component.scss [hunk 1]
```diff
@@ -1,38 +1,70 @@
...
-    //Forcing clarity ui inputs to full width of the screen
-    align-items: initial;
-  }
 
-  .clr-input {
+  input, textarea {
+    border: none;
+    border-radius: 6px;
+    box-shadow: 5px 5px black;
+    box-shadow: 0px 3px 6px #0000001A;
```
Comment (MaxKorlaar) [inline src/app/modules/project/add/manual/manual.component.scss:27]: These properties just override each other.
```suggestion
    box-shadow: 0px 3px 6px #0000001A;
```

File: src/app/modules/project/add/source/source.component.html [hunk 2]
```diff
@@ -1,75 +1,67 @@
...
+<div class="row">
+  <div class="col-12 text-center">
     <a routerLink="/project/add/manual">Add Project Manual</a>
   </div>
 </div>
-<div class="clr-row justify-content-center">
-  <div class="clr-col-4 clr-offset-4">
+<div class="row">
+  <div class="col-4 offset-4">
     <p>Note: Only add a project manual if the source of your project does not exist in the list above.</p>
```
Comment (MaxKorlaar) [inline src/app/modules/project/add/source/source.component.html:65]:
```suggestion
    <p>Note: Only manually add a project if the source of your project is not listed above.</p>
```

File: src/app/modules/project/add/source/source.component.scss [hunk 3]
```diff
@@ -41,21 +40,49 @@ p {
...
 .source {
-  margin-top: 20px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  color: #fff;
+  line-height:100%;
+  height:100%;
```
Comment (MaxKorlaar) [inline src/app/modules/project/add/source/source.component.scss:62]: Spaces between property name and value are preferred for clarity.
```suggestion
  line-height: 100%;
  height: 100%;
```

File: src/app/modules/project/overview/overview.component.scss [hunk 4]
```diff
@@ -152,3 +152,13 @@
     padding: 10px;
   }
 }
+
+accordion-group .card-header {
+  background: red!important;
```
Comment (MaxKorlaar) [inline src/app/modules/project/overview/overview.component.scss:157]: We really shouldn't use `!important`. Can this be overridden?

File: src/app/modules/highlight/top-highlight-cards/top-highlight-cards.component.html [hunk 5]
```diff
@@ -1,37 +1,37 @@
-<!-- 
-  
+<!--
+
   Digital Excellence Copyright (C) 2020 Brend Smits
```
[the remark this answers is not in the document]
Comment (Brend-Smits) [reply src/app/modules/highlight/top-highlight-cards/top-highlight-cards.component.html:3]: Good point, for now, keep it as it is. I will contact some people to see if we can actually use Digital Excellence as the name.
Also, can you link the feature in PHPStorm, this sounds interesting.

File: angular.json [hunk 6]
```diff
@@ -43,14 +43,13 @@
             "tsConfig": "tsconfig.app.json",
             "aot": true,
             "assets": ["src/favicon.ico", "src/assets"],
-            "styles": ["node_modules/@clr/icons/clr-icons.min.css", "node_modules/@clr/ui/clr-ui.min.css", "src/styles.scss"],
+            "styles": ["node_modules/bootstrap/dist/css/bootstrap.min.css", "src/styles.scss"],
```
[the remark this answers is not in the document]
Comment (Brend-Smits) [reply angular.json:46]: What do you recommend? Using the bootstrap.scss instead?

File: src/app/modules/highlight/top-highlight-cards/top-highlight-cards.component.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (MaxKorlaar) [reply src/app/modules/highlight/top-highlight-cards/top-highlight-cards.component.html:3]: https://www.jetbrains.com/help/phpstorm/copyright.html#copyright-profile

[the remark this answers is not in the document]
Comment (Brend-Smits) [reply src/app/modules/highlight/top-highlight-cards/top-highlight-cards.component.html:3]: That's cool, thanks.

File: angular.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (MaxKorlaar) [reply angular.json:46]: Yass https://getbootstrap.com/docs/4.1/getting-started/theming/#importing

[the remark this answers is not in the document]
Comment (Brend-Smits) [reply angular.json:46]: I see, that's nice. @RensNooren this definitely needs changing in that case.

File: src/app/modules/project/overview/overview.component.scss (hunk 4 above)
Comment (RensNooren) [reply src/app/modules/project/overview/overview.component.scss:157]: This is all going to be fixed in "Fix: Accordion styling #106"

File: angular.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (Brend-Smits) [reply angular.json]: This should be fixed now

File: angular.json [hunk 7]
```diff
@@ -43,14 +43,13 @@
             "tsConfig": "tsconfig.app.json",
             "aot": true,
             "assets": ["src/favicon.ico", "src/assets"],
-            "styles": ["node_modules/@clr/icons/clr-icons.min.css", "node_modules/@clr/ui/clr-ui.min.css", "src/styles.scss"],
+            "styles": ["node_modules/bootstrap/scss/bootstrap.scss", "src/styles.scss"],
```
Comment (MaxKorlaar) [inline angular.json:46]: Can we use [functions](https://getbootstrap.com/docs/4.0/getting-started/theming/#functions) when importing the scss like this, or do we need to import it in our own stylesheet?

File: src/app/components/app-layout/app-layout.component.html [hunk 8]
```diff
@@ -33,21 +33,31 @@
...
+        <button id="button-basic" dropdownToggle type="button" class="btn btn-primary dropdown-toggle"
+                aria-controls="dropdown-basic">
+          {{ name }} <img class="profile-picture" src="assets/janette-doe.png"> <span class="caret"></span>
+        </button>
+        <ul id="dropdown-basic" *dropdownMenu class="dropdown-menu"
+            role="menu" aria-labelledby="button-basic">
+          <li role="menuitem"><a class="dropdown-item" href="#">My profile</a></li>
+          <li role="menuitem"><a class="dropdown-item" href="#">Account settings</a></li>
+          <li class="divider dropdown-divider"></li>
+          <li role="menuitem"><a class="dropdown-item" *ngIf="isAuthenticated" (click)="signout()" routerLinkActive="active">Sign Out</a>
```
Comment (MaxKorlaar) [inline src/app/components/app-layout/app-layout.component.html:48]: This check is redundant. The parent div at line 37 is rendered for the same condition.
```suggestion
          <li role="menuitem"><a class="dropdown-item" (click)="signout()" routerLinkActive="active">Sign Out</a>
```

File: src/app/components/app-layout/app-layout.component.scss [hunk 9]
```diff
@@ -115,3 +127,38 @@
...
+    cursor: pointer;
+  }
+}
+
+.btn-primary {
+  .show { }
+  .dropdown-toggle {
+    &, &:focus { }
+  }
+  // etc
```
Comment (MaxKorlaar) [inline src/app/components/app-layout/app-layout.component.scss:145]: My suggestion was an example, has this been tested? If it works just like this, these empty statements can be removed.

File: angular.json (hunk 7 above)
Comment (Brend-Smits) [reply angular.json:46]: Good question, I only tested if styling worked as expected. I have little experience with SCSS so this is hard for me to test, could you or @RensNooren perhaps test this?

**Pull request merged**